pilot/cmd/pilot-agent/main.go

```diff
@@ -417,6 +424,9 @@ func init() {
 		"Port on which Envoy should listen for administrative commands")
 	proxyCmd.PersistentFlags().StringVar(&controlPlaneAuthPolicy, "controlPlaneAuthPolicy",
 		values.ControlPlaneAuthPolicy.String(), "Control Plane Authentication Policy")
+	proxyCmd.PersistentFlags().StringVar(&appHTTPProbers, status.KubeAppProberCmdFlagName, "",
```

Comment:
I donot think pass a json string as a flag is easy to use.

Reply:
How about make this a structured array like 
struct{
    containerName string
    liveness          
    readiness
}

Reply:
I think this one is resolved because the flag is auto-injected and not intended to be set by human?

Reply:
make sense